sdk/avalon_sdk/connector/direct/jrpc/jrpc_worker_registry.py

```diff
@@ -94,7 +97,21 @@ def worker_lookup(self, worker_type=None, organization_id=None,
         }
 
         if worker_type is not None:
-            json_rpc_request["params"]["workerType"] = worker_type.value
+            if isinstance(worker_type, WorkerType):
+                json_rpc_request["params"]["workerType"] = worker_type.value
+            elif isinstance(worker_type, int):
+                json_rpc_request["params"]["workerType"] = worker_type
+            else:
+                return {
```

Comment:
can we use `create_error_response()`
https://github.com/hyperledger/avalon/blob/master/common/python/utility/jrpc_utility.py#L20